nginx/proxy-files.conf

```diff
@@ -0,0 +1,18 @@
+server {
+  listen 443 ssl;
+  server_name ${FILE_DOMAIN};
+
+  ssl_certificate /etc/nginx/ssl.crt;
+  ssl_certificate_key /etc/nginx/ssl.key;
+
+  location / {
+    root /data/files;
+    index index.html;
+    try_files $uri $uri/ /index.html;
+  }
+
+  # Redirect HTTP to HTTPS
```

Comment:
I don't think this is needed because we're only exposing SSL port here https://github.com/aesirxio/web3-id-preregistration-backend/pull/58/files#diff-8d2116c39eb8cc36d711c99b58f2e03242375324de602d4a2c8023c83c75985bR2